pyfmt/__init__.py

```diff
@@ -25,8 +27,121 @@
 ]
 
 
-def pyfmt(path, check=False, line_length=100, extra_isort_args="", extra_black_args="") -> int:
+def pyfmt_title():
+    """
+    Display a cool ascii art title
+    """
+    pyfmt_splash = [
+        "               __           _    ",
+        "              / _|         | |   ",
+        "  _ __  _   _| |_ _ __ ___ | |_  ",
+        " | '_ \| | | |  _| '_ ` _ \| __| ",
+        " | |_) | |_| | | | | | | | | |_  ",
+        " | .__/ \__, |_| |_| |_| |_|\__| ",
+        " | |     __/ |                   ",
+        " |_|    |___/                    ",
+    ]
+    print("\033[94m")
+    for part in pyfmt_splash:
+        print(f"{part.center(os.get_terminal_size().columns, ' ')}")
+    print("\033[0m")
+
+
+def find_all_files_and_dirs():
+    """
+    Map out all files and directories in the current
+    working directory
+    """
+    all_files = []
+    all_dirs = []
+    for root, dirs, files in os.walk("."):
+        for name in files:
+            # Saving filename
+            all_files.append(name)
+        for name in dirs:
+            # Saving directory name
+            all_dirs.append(os.path.abspath(os.path.join(root, name)))
+    # Remove duplicates
+    all_files = list(set(all_files))
+
+    return all_files, all_dirs
+
+
+def display_divider(title="", character="=", color_code="\033[94m"):
+    """
+    Divider between sections of program.
+    """
+    if title:
+        print(
+            "\n"
+            + "{}".format(color_code)
+            + "  {}  ".format(title.upper()).center(
+                os.get_terminal_size().columns, "{}".format(character)
+            )
+            + "\033[0m"
+        )
+    else:
+        print(
+            "\n"
+            + "{}".format(color_code)
+            + "".center(os.get_terminal_size().columns, "{}".format(character))
+            + "\033[0m"
+        )
+
+
+def pyfmt(
+    path, skip="", check=False, line_length=100, extra_isort_args="", extra_black_args=""
+) -> int:
     """Run isort and black with the given params and print the results."""
+    pyfmt_title()  # Display title
+    timer_start = time.time()  # Measure how long everything takes
+
+    if skip:
+        # Map out current working directory
+        all_files, all_dirs = find_all_files_and_dirs()
+        # If specified files and directories exist, store the filenames
+        skips = skip.split(",")
+        filenames_to_skip = []
+        for item in skips:
+            if item in all_files:
+                if item.split(".")[-1] == "py":
+                    filenames_to_skip.extend([item])
+            elif os.path.abspath(item) in all_dirs:
+                # Saving all filenames in directory
+                files_in_dir = list()
```

Comment:
```suggestion
                files_in_dir = []
```

using `[]` is slightly more efficient than `list()` 🙂